### PR TITLE
Add prometheus-api-client scenario to the AI/ML Course

### PR DESCRIPTION
**Pull request by 4n4nd**
This PR will add a new pathway for AI/ML course.
This uses the Openshift 3.11 base image. I tested it with 4.2 but the env set up times were considerably faster in 3.11.

This scenario can be tested here: https://www.katacoda.com/4n4nd/scenarios/prometheus-api-client

### CODE REVIEW

**Comment by BenHall** [comment]
This looks good to me. Shall I merge @jankleinert ?

**Comment by 4n4nd** [comment]
Hey, I added one more commit to add some images in the steps.

File: ai-machine-learning/prometheus-api-client/02-loading-the-notebook-templates.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+http://prometheus-demo-route-myproject.[[HOST_SUBDOMAIN]]-80-[[KATACODA_HOST]].environments.katacoda.com/
+
+![Prometheus Console](./prometheus-api-client/assets/02-prometheus-console.png)
+
+The jupyter environment with the workshop notebooks is available here:
+
+http://custom-notebook-myproject.[[HOST_SUBDOMAIN]]-80-[[KATACODA_HOST]].environments.katacoda.com/
+
+You will need a password to access the environment.
+The password is `secret`{{copy}}
```
Comment (joshix) [inline ai-machine-learning/prometheus-api-client/02-loading-the-notebook-templates.md]: Suggest reconciling with the OpenShift console credentials used below, if possible.

Comment (4n4nd) [reply ai-machine-learning/prometheus-api-client/02-loading-the-notebook-templates.md]: Just wanted to keep it consistent with the [other](https://github.com/openshift-labs/learn-katacoda/blob/master/ai-machine-learning/jupyter-notebooks/04-deploying-the-jupyter-notebook.md ) [notebook](https://github.com/openshift-labs/learn-katacoda/blob/master/ai-machine-learning/jupyter-workspace/04-deploying-the-jupyter-workspace.md) [scenarios](https://github.com/openshift-labs/learn-katacoda/blob/master/ai-machine-learning/custom-notebooks/05-deploying-the-notebook-image.md).

Comment (joshix) [reply ai-machine-learning/prometheus-api-client/02-loading-the-notebook-templates.md]: Right on -- I think this is a configuration element set earlier in the content, now that I see the context.

**Pull request merged**